pom.xml

```diff
@@ -37,10 +44,8 @@
     <version.assertj>3.5.2</version.assertj>
     <version.jcip.annotations>1.0</version.jcip.annotations>
     <version.shrinkwrap.resolver>2.2.5</version.shrinkwrap.resolver>
+    <version.snakeyaml>1.17</version.snakeyaml>
```

Comment:
And it's still done this way for a reason I guess. To not pollute user's classpath with our internals.